Garhal/Garhal.vcxproj

```diff
@@ -76,38 +60,6 @@
     <DriverType>KMDF</DriverType>
     <DriverTargetPlatform>Universal</DriverTargetPlatform>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|ARM'" Label="Configuration">
```

Comment:
Can't really agree with this part of the code

Reply:
Maybe I don't understand something, but what's the point of building a driver for ARM architecture that CSGO does not support.

Reply:
I meant the windows SDK settings. Not sure why it is tagging the ARM line on github.

Reply:
Sorry i really do not understand, Garhal.vcxproj uses latest available Windows SDK version. It's default behaviour if version not set by developer. Everything builds on the latest version of Visual Studio with the latest version of the WDK.